metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -16,10 +17,13 @@
       type: keyword
       description: >
         Name of Kibana's internal index
+
+    # What does this match to?
     - name: host.name
```

Comment:
Not sure I understand the question but I can tell you what type of values can be expected in this field. I did some testing and it looks like this field can contain a hostname or an IP address. Does that help?

Reply:
Is this the hostname of the host machine on which Kibana is running on?

Reply:
Not exactly, this is whatever the user sets as the value of the `server.host` setting in `kibana.yml`. Kibana will bind it's HTTP server to this value. Here's the relevant comments for this setting from `kibana.yml`:

```
# Specifies the address to which the Kibana server will bind. IP addresses and host names are both valid values.
# The default is 'localhost', which usually means remote machines will not be able to connect.
# To allow connections from remote users, set this parameter to a non-loopback address.
#server.host: "localhost"
```

Reply:
Changed to `service.address` in https://github.com/elastic/beats/pull/10350.